package.json

```diff
@@ -27,14 +29,23 @@
   },
   "devDependencies": {
     "@babel/core": "^7.14.3",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-react": "^7.14.5",
+    "@types/enzyme": "^3.10.8",
+    "@types/jest": "^26.0.23",
     "@types/plotly.js": "^1.54.10",
     "@types/react": "^17.0.9",
     "@types/react-dom": "^17.0.6",
     "@types/react-router-dom": "^5.1.7",
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
+    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
```

Comment:
Can we avoid the use of this library? It looks unofficial and not so popular library.

Reply:
I asked my friend who is a frontend engineer, if I should use Enzyme on optuna-dashboard. That's because Enzyme doesn't support React v17, that was released about six months ago.

He recommended me to use https://github.com/testing-library/react-testing-library instead of Enzyme. Enzyme hasn't been able to keep up with the release cycle of React.js. So could you remove Enzyme from the dependencies and use react-testing-library instead?

Reply:
Yes, I was first using Enzyme as I didn't pull the latest change, which upgrades React from 16 to 17. Then I realized that Enzyme hasn't supported 17 yet. So I have to find some unofficial adapter to replace the older one. Don't worry, I'll check if using react-testing-library will be better (which I believe will be the case)

Reply:
Just did some research on https://github.com/testing-library/react-testing-library and found that their support for `<table>` element is poor, so to select a whole column requires some hacky operations as react-testing-library does not encourage users to detect props change.